docs/README.md

```diff
@@ -1,22 +1,34 @@
-- Service Mesh for service-to-service communication and proxying
-- Service management
-- Secure API exposure
-- In-cluster observability
-- CLI supported by the intuitive UI
+### Kyma modules 
 
-Read about [Kyma main areas](./01-overview/main-areas) to learn more.
+> **NOTE:** The entries marked with "*" are still components that will be modularized soon.
 
-Kyma is built upon leading cloud-native, open-source projects, such as Istio, NATS, Serverless, and Prometheus. The features developed by Kyma are the unique “glue” that holds them together, so you can connect and extend your applications easily and intuitively. To learn how to do that, head over to the [Get Started](./02-get-started) section where you can find step-by-step instructions to get your environment up and running.
+| Module | Purpose |
+|---|---|
+| [BTP Operator]((https://kyma-docs.netlify.app//?basePath=https://raw.githubusercontent.com/kyma-project/btp-manager/aa50848013372806eaf2e707c217b8bed4eb09cb/docs/user/&homepage=README.md&sidebar=true&loadSidebar=_sidebar.md&browser-tab-title=BTP%20Operator%20Documentation#/)) |Within the BTP Operator module, BTP Manager installs SAP BTP Service Operator that allows you to consume SAP BTP services from your Kubernetes cluster using Kubernetes-native tools. |  |
+| [Istio*](https://github.com/kyma-project/istio) | Istio is a service mesh with Kyma-specific configuration. | |
+| [Serverless](https://github.com/kyma-project/serverless-manager) | With the Serverless module, you can define simple code snippets (Funtions) with minimal implementation effort. |  |
+| [Telemetry*](https://github.com/kyma-project/telemetry-manager) | Enable telemetry agents to easily collect application logs and distributed traces for your application and dispatch them to backends.|   |
+| [Eventing*](https://github.com/kyma-project/eventing-manager) | Eventing provides functionality to publish and subscribe to CloudEvents. <br> At the moment, the SAP Event Mesh default plan and NATS (provided by the NATS module) are supported. |  |
+| [NATS*](https://github.com/kyma-project/nats-manager) |NATS deploys a NATS cluster within the Kyma cluster. You can use it as a backend for Kyma Eventing. |  |
+| [Application Connector*](https://github.com/kyma-project/application-connector-manager) | Application Connector allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process doesn't change, which allows you to avoid any configuration or network-related problems. | 
+| API Gateway* |API Gateway provides functionalities that allow you to expose and secure APIs. |
 
-The extensions and customizations you create are decoupled from your core applications, which adds to these general advantages of using Kyma:
+### Kyma Beta modules:
+> **NOTE:** The entries marked with "*" are still components that will be modularized soon.
```

Comment:
Do we need the NOTE? There are no components in the table.

Reply:
> Do we need the NOTE? There are no components in the table.

Just forgot to remove it. Thanks for spotting it :)